### PR TITLE
Version 1.9.8

### PR DESCRIPTION
**Pull request by Krakinou**
## Problem
- Upgrade to 1.9.8

### CODE REVIEW

**Comment by Krakinou** [comment]
!testme

**Comment by Krakinou** [comment]
!testme

**Comment by Krakinou** [comment]
!testme

File: conf/app.src.default [hunk 1]
```diff
@@ -0,0 +1,5 @@
+SOURCE_URL=https://buildbot.libretro.com/stable/__UPSTREAM_VERSION__/emscripten/RetroArch.7z
+SOURCE_SUM=03166297e546e44755f1bacd8a4bbc90b58fae746864c3f67eaadd8c814906b2
```
Comment (alexAubin) [inline conf/app.src.default:2]: Eeeeh it works I guess, but you'll end up having to update the checksum each time the version changes anyway soooooo ? :thinking:

**Review by alexAubin — APPROVED** [review]
LGTM

**Review by alexAubin — CHANGES_REQUESTED** [review]
In fact it's broken precisely because of the weird app.src stuff: 

```
25971 WARNING https://buildbot.libretro.com/stable/1.9.8~ynh1/emscripten/RetroArch.7z:
25973 WARNING 2021-08-26 08:21:04 ERROR 404: Not Found.
```

Gotta remove the ~ynh1 part

File: conf/app.src.default (hunk 1 above)
Comment (Krakinou) [reply conf/app.src.default:2]: yep, but I'm tired to update the version number at different places as I always forget one, so I tried to "centralized" it a bit :)
The things is: it worked on my local test but not on the package_check, so I obviously made some stupid mistake somewhere

**Comment by Krakinou** [comment]
!testme

**Comment by Krakinou** [comment]
!testme

**Comment by Krakinou** [comment]
!testme

**Review by alexAubin — APPROVED** [review]

**Pull request merged**